DDP-5815: fix fields alignment, font-size and label

aglushko-broad opened the pull request:
before:
![image](https://user-images.githubusercontent.com/82094572/114687253-3e9c2e00-9d1c-11eb-86c4-b8ac0bcd7216.png)

after:
![image](https://user-images.githubusercontent.com/82094572/115042221-f3c91480-9edb-11eb-86d5-ef28f9447ea5.png)

Commit:
DDP-5815: fix fields alignment, font-size and label

MocanaAtBroad commented on ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/dropdownActivityPicklistQuestion.component.ts:
```diff
@@ -9,54 +9,60 @@ import { NGXTranslateService } from '../../../services/internationalization/ngxT
+              </span>{{ questionIdToCharacterLeftMsg[details.stableId] }}
+          </p>
+      </ng-container>
+    </mat-form-field>`,
```
@aglushko-broad : unfortunately git diff is not really helping to narrow down the change you made here. Could you briefly highlight/explain?

aglushko-broad replied:
sure! the changes I made:
- wrap with mat-form-field and assign matNativeControl to native select in order to apply material styles for alignment
- remove the first option from native select which was equal to the label and add mat-label (I had to add it out of native select ng-template in order to let internal mechanism which uses ng-content to handle it)

alextanlan commented:
You improved the alignment.
However, `ddp-activity-answer`-s are still not on the same level.

For testing it  - I moved left block to the right as to overlap them (by adding `position: relative; left: 50px`)

- Before changes (on dev env):

![before](https://user-images.githubusercontent.com/7396837/114744451-627c6580-9d56-11eb-8981-27e0e668fa24.png)

- After changes (run your branch locally):

![after](https://user-images.githubusercontent.com/7396837/114744500-6f00be00-9d56-11eb-85aa-22cbacddd7ce.png)

Commit:
DDP-5815 fix alignment

aglushko-broad commented:
> You improved the alignment.
> However, `ddp-activity-answer`-s are still not on the same level.
> 
> For testing it - I moved left block to the right as to overlap them (by adding `position: relative; left: 50px`)
> 
> * Before changes (on dev env):
> 
> ![before](https://user-images.githubusercontent.com/7396837/114744451-627c6580-9d56-11eb-8981-27e0e668fa24.png)
> 
> * After changes (run your branch locally):
> 
> ![after](https://user-images.githubusercontent.com/7396837/114744500-6f00be00-9d56-11eb-85aa-22cbacddd7ce.png)

nice catch! didn't see it on the small screen. Fixed

alextanlan commented:
Perfect!

MocanaAtBroad commented:
OK. One super minor thing: the bottom of the "g" in the words "age" and "diagnosis" is getting cut off (it shows up in your screen capture too):
![Screen Shot 2021-04-15 at 12 42 57 PM](https://user-images.githubusercontent.com/29984380/114906785-65c03180-9de8-11eb-921b-68fc983919dd.png)

MocanaAtBroad commented:
Another thing I am noticing is that after introducing the Material field label, this particular question looks unbalanced (Age of diagnosis has Material label, but Cancer Type just placeholder). Looking if this something we can tweak from server configuration....

MocanaAtBroad commented:
Let's back off/undo the change for Material label for the pull-down.

I think it might have wider repercussions/used in another apps and each of them would have to be verified that it looks good. Even in this case, just looking at code and configuration it is not totally clear what is the right thing to do now with the cancer type field while at the same time ensuring that it continuous to be right in all our apps.

Let's just make sure the alignment/original bug reported is right. We need that. At some point we can come back and revisit the Material label problem.

Commit:
DDP-5816 fix label floating

aglushko-broad commented:
> Let's back off/undo the change for Material label for the pull-down.
> 
> I think it might have wider repercussions/used in another apps and each of them would have to be verified that it looks good. Even in this case, just looking at code and configuration it is not totally clear what is the right thing to do now with the cancer type field while at the same time ensuring that it continuous to be right in all our apps.
> 
> Let's just make sure the alignment/original bug reported is right. We need that. At some point we can come back and revisit the Material label problem.

thanks for noticing that label position for select doesn't match the position for autocomplete input! I fixed it quickly, you can take a look again. Though I see your concern that it affects all apps I'd say having the label is better than don't have a label at all, it's more noticeable in the selected state, then the user can't be sure what exactly s/he selected before.
This is how it looks now
![image](https://user-images.githubusercontent.com/82094572/114981203-4bdd2800-9e96-11eb-9749-5187086d7695.png)
But anyway it's not a problem to remove the label for now and return it how it was before. Could you please confirm one more time we want to do it, thanks!

Commit:
DDP-5815 remove label

aglushko-broad commented:
> Let's back off/undo the change for Material label for the pull-down.
> 
> I think it might have wider repercussions/used in another apps and each of them would have to be verified that it looks good. Even in this case, just looking at code and configuration it is not totally clear what is the right thing to do now with the cancer type field while at the same time ensuring that it continuous to be right in all our apps.
> 
> Let's just make sure the alignment/original bug reported is right. We need that. At some point we can come back and revisit the Material label problem.

@MocanaAtBroad  it's all done! I updated the screenshot in the PR description

MocanaAtBroad reviewed (commented):
Looks good! Let's merge.
